third_party/rust/byteorder/v1/BUILD.gn

```diff
@@ -33,8 +33,5 @@ cargo_crate("lib") {
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
   proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
   proc_macro_configs += [ "//build/config/compiler:no_chromium_code" ]
-  features = [
-    "i128",
-    "std",
-  ]
+  features = [ "std" ]
```

Comment:
This change was made by `gnrt`. I looked through the various references and couldn't find a matching specification for the `i128` feature, so I suspect this is correct. Normally we need 128-bit support for crypto token serialization, so maybe we were using this at one point and then switched to a different approach?